<?xml version="1.0" encoding="UTF-8"?>

<launch>
  <arg name="node_name" default="mosaic_gnss" />
  <arg name="param_file_name" />
  <arg name="output" default="screen" />
  <arg name="respawn" default="false" />
  <arg name="clear_params" default="true" />

  <node pkg="rosaic" type="rosaic_node" name="$(arg node_name)"
        output="$(arg output)" 
        clear_params="$(arg clear_params)"
        respawn="$(arg respawn)">
    <rosparam command="load" 
              file="$(find rosaic)/config/$(arg param_file_name).yaml" />
  </node>
</launch>
